lib/serializer/ILICReader.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * ILICReader.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+#include  "JsonTreeSerializer.h"
+
+VCMI_LIB_NAMESPACE_BEGIN
+
+class DLL_LINKAGE ILICReader
+{
+protected:
+	void readLICPart(const JsonNode & part, const JsonSerializeFormat::TDecoder & decoder, const bool val, std::vector<bool> & value) const;
```

Comment:
why `const bool` param?

Reply:
@kambala-decapitator because it was in both JsonUpdater.cpp + JsonSerializer.cpp. I can remove const if desired.

Reply:
@kambala-decapitator `const bool` parameter changed to `bool`. But it is still `const bool` semantically, because it should not be changed.

Reply:
at the calling site it wouldn't change anyway. tbh I've never seen declaring POD type params as `const`.